src/components/DayPickerRangeController.jsx

```diff
@@ -565,6 +570,7 @@ export default class DayPickerRangeController extends React.PureComponent {
 
         // eslint-disable-next-line react/destructuring-assignment
         if (this.state.dateOffset && this.state.dateOffset.start && this.state.dateOffset.end) {
+          // eslint-disable-next-line react/destructuring-assignment
```

Comment:
is this override needed?

Reply:
We are silencing eslint on the line above so it made sense to me to also silence this line.

Reply:
Your call on whether we keep this.

Reply:
It should only be kept if it's needed to silence a warning; if removing it results in no warning, then remove it.

Reply:
🤔 at least for me, it removed the warning for the line below

Reply:
in that case, keep it